src/index/IndexImpl.h

```diff
@@ -366,7 +372,12 @@ class IndexImpl {
    *       actual wordId range.
    *       TODO: improve size estimate by adding a correction factor.
    */
-  size_t getSizeOfTextBlocks(const string& word, bool forWord) const;
+  size_t getSizeOfTextBlocksSum(const std::string& word,
+                                TextScanMode textScanMode) const;
+
+  static size_t getSizeOfTextBlocksSum(
```

Comment:
short comment: "same as the overload above, but works on metadata objects instead of a word or prefix.